site/content/blog/2019-06-05-ldap-nested-groups-modelling-and-representation-in-code.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "LDAP Nested Groups: Modelling and Representation in Code"
+author: Martin Kraft
+date: 2019-06-05T12:00:00-04:00
+github: mkraft
+community: martin.kraft
+---
+
+## LDAP Group Sync in Mattermost
+
+In [Mattermost](https://mattermost.com) v5.8 we deployed the [LDAP group sync feature](https://docs.mattermost.com/deployment/ldap-group-sync.html) to allow Enterprise Edition customers to create and synchronize groups in Mattermost matching their LDAP groups. The goal was to ease onboarding by automatically adding group members to configured teams and channels. 
```

Comment:
```suggestion
In [Mattermost](https://mattermost.com) v5.8 we deployed [LDAP group sync feature](https://mattermost.com/pl/default-ldap-group-sync) to enable Enterprise Edition customers to create and synchronize groups in Mattermost matching their LDAP groups. The goal was to ease onboarding by automatically adding group members to configured teams and channels. 
```

Reply:
@jasonblais Was removing the word "the" intentional in this suggestion or was it a typo?

"In Mattermost v5.8 we deployed the LDAP group sync feature to allow..."
"In Mattermost v5.8 we deployed LDAP group sync feature to allow..."